### DISCUSSION TITLE
404 error when using include statement

### DISCUSSION THREAD

**Discussion by MML1357** [prisma/prisma#7698] [Q&A]
Hi.
I'm trying to use a include statement inside a findMany but I'm getting a 404 error when I go to the URL that is suppose to show the query result.
This only occurs with include. Any where or select works perfectly.
This is my sample code:

const query = await prisma.evento.findMany({
    include: {
     calificacion_evento: true
    }
})

The models looks like:

model evento {
 id                    BigInt                @id @default(autoincrement())
 codigo                String                @db.VarChar
 detalle               String
 calificacion_evento   calificacion_evento[]
}
model calificacion_evento {
  id                 BigInt       @id @default(autoincrement())
  evento_id          BigInt
  fecha_creacion     DateTime     @db.Timestamptz(6)
  comentario         String?
  evento             evento       @relation(fields: [evento_id], references: [id])
}

I'm using tsoa to document my proyect with OpenAPI.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
